src/components/side_panel/chart/chart_with_axis/design_panel.ts

```diff
@@ -2,6 +2,7 @@ import { Component, useState } from "@odoo/owl";
 import { getColorsPalette, getNthColor, setColorAlpha, toHex } from "../../../../helpers";
```

Comment:
nitpick: in commit message

fixed to the minimum value **between** 10 and 1 **below the** number of